docs/pages/includes/device-spec.mdx

```diff
@@ -0,0 +1,51 @@
+```yaml
+kind: device
+version: v1
+metadata:
+  # name is a unique random identifier for the device in the cluster.
+  name: 5ff09619-527c-4a17-973f-cd9cd5c93990
+spec:
+  # asset_tag is a user controlled identifier for the device. This is usually a device serial number.
+  asset_tag: HD6M74XNCK
+
+  # collected_data is a list of data recently collected from the device.
+  # Each list item is an object containing event and device metadata..
+  collected_data:
+    # collect_time is the device-recorded timestamp of when the data was collected.
+  - collect_time:
+      nanos: 312862000
+      seconds: 1677099866
```

Comment:
Do you think we should convert this timestamp too, as we do the others? (Follow up for another PR, most likely.)